webpack.config.dev.js

```diff
@@ -1,17 +1,29 @@
 const merge = require('webpack-merge');
-const LiveReloadPlugin = require('webpack-livereload-plugin');
 const path = require('path');
 
 const devConfig = require('@centreon/frontend-core/webpack/patch/dev');
 const baseConfig = require('./webpack.config');
 
+const devServerPort = 9090;
+
 module.exports = merge(baseConfig, devConfig, {
+  output: {
+    publicPath: `http://localhost:${devServerPort}/static/`,
```

Comment:
Does it work if browser is not on the same machine than the build

Reply:
It works even if the index.html file is served remotely, but the dev server must run locally.